src/app/content/components/BookBanner.tsx

```diff
@@ -176,158 +176,144 @@ export const BarWrapper = styled.div<BarWrapperProps>`
   ${ifMiniNav(`margin-top: -${bookBannerDesktopMiniHeight}rem`)}
 `;
 
-export interface PropTypes {
-  pageNode?: ArchiveTreeSection;
-  book?: Book;
-  hasUnsavedHighlight?: boolean;
-  bookTheme: BookWithOSWebData['theme'];
-}
-
 interface BookBannerState {
   scrollTransition: boolean;
-  tabbableBanner: 'mini' | 'big';
+  tabbableBanner: 'big' | 'mini';
```

Comment:
lol like this doesn't matter but please try to make an effort not to make meaningless changes, it makes the diff harder to read.